k8s/config/samples/k8s_v1alpha1_sidecar.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: k8s.v6d.io/v1alpha1
+kind: Sidecar
+metadata:
+  name: sidecar-sample
+spec:
+  #replicas: 1
+  image: test
```

Comment:
Looks wired. What is "test"?

Reply:
Ummm, sorry about it. It's only for testing, I will add a complete cr next.